### PR TITLE
docs: add auth0 rule documentation

### PR DESCRIPTION
**Pull request by Jameskmonger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,15 @@ See "Environment variables" above for info on how to store them.
   - **domain** as `AUTH0_DOMAIN`
   - **client id** as `AUTH0_SPA_CLIENT_ID`
 
+- Add a new Empty Rule to Auth0
+
+  This is to check to see whether the incoming user has meta data set against their User in Auth0.
```
Comment (Jameskmonger) [inline README.md:78]:
```suggestion
  This exposes the user's Auth0 metadata.
```

File: .auth0/rules/rule.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/*
```
Comment (Jameskmonger) [inline .auth0/rules/rule.js:1]: Please rename file to `attach-metadata.js`

Comment (Jameskmonger) [reply .auth0/rules/rule.js:1]: and run prettier

File: README.md (hunk 1 above)
Comment (ollymonger) [reply README.md]: sorted

File: .auth0/rules/rule.js (hunk 2 above)
Comment (ollymonger) [reply .auth0/rules/rule.js:1]: done!

File: README.md [hunk 3]
```diff
@@ -73,6 +73,15 @@ See "Environment variables" above for info on how to store them.
...
   - **client id** as `AUTH0_SPA_CLIENT_ID`
 
+- Add a new Empty Rule to Auth0
+
+  This exposes the user's Auth0 metadata.
+
+  - Go to your Auth0 Portal
+  - Find: Auth Pipeline > Rules in the left side navigation.
+  - Create a new Empty Rule
+  - Copy and paste the contents of: .auth0/rules/rule.js into this section.
```
Comment (Jameskmonger) [inline README.md:83]:
```suggestion
  - Copy and paste the contents of: `.auth0/rules/attach-metadata.js` into this section.
```

**Pull request merged**